olga_poem.md

```diff
@@ -0,0 +1,34 @@
+In the realm of code, where developers thrive,
+A journey awaits, for you to dive.
+Into the world of Git and its hub,
+A poem for learning GitHub, a techy love.
+
+With branches and commits, we start our quest,
+To collaborate and be our very best.
+GitHub, the platform where coding hearts meet,
+Empowering us to create, innovate, and compete.
+
+Clone repositories, bring projects to life,
+Version control, easing the strife.
+Fork and pull, a community's dance,
+Sharing knowledge, giving code a chance.
+
+Pull requests, a gateway for review,
+Collaborators unite, as ideas accrue.
+Issues and milestones, tracking our goals,
+Resolving conflicts, harmonizing souls.
+
+From READMEs to wikis, documentation prevails,
```

Comment:
I just dont know if README should be ReadME or readme or README. 

I like ReadMe best.